edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/CombinedKubernetesConfigProvider.cs

```diff
@@ -108,5 +108,132 @@ static string BindPath(Uri uri)
+    //
+    //     public CombinedKubernetesConfigProvider(IEnumerable<AuthConfig> authConfigs, IConfigSource configSource)
+    //         : base(authConfigs)
+    //     {
+    //         this.configSource = Preconditions.CheckNotNull(configSource, nameof(configSource));
+    //     }
+    //
+    //     static CreateContainerParameters CloneOrCreateParams(CreateContainerParameters createOptions) =>
+    //         createOptions != null
+    //             ? JsonConvert.DeserializeObject<CreateContainerParameters>(JsonConvert.SerializeObject(createOptions))
+    //             : new CreateContainerParameters();
+    //
+    //     public override CombinedDockerConfig GetCombinedConfig(IModule module, IRuntimeInfo runtimeInfo)
+    //     {
+    //         CombinedDockerConfig combinedConfig = base.GetCombinedConfig(module, runtimeInfo);
+    //
+    //         // if the workload URI is a Unix domain socket then volume mount it into the container
+    //         CreateContainerParameters createOptions = CloneOrCreateParams(combinedConfig.CreateOptions);
+    //         this.MountSockets(module, createOptions);
+    //         this.InjectNetworkAliases(module, createOptions);
+    //
+    //         return new CombinedDockerConfig(combinedConfig.Image, createOptions, combinedConfig.AuthConfig);
+    //     }
+    //
+    //     void InjectNetworkAliases(IModule module, CreateContainerParameters createOptions)
+    //     {
+    //         if (createOptions.NetworkingConfig?.EndpointsConfig == null)
+    //         {
+    //             string networkId = this.configSource.Configuration.GetValue<string>(Core.Constants.NetworkIdKey);
+    //             string edgeDeviceHostName = this.configSource.Configuration.GetValue<string>(Core.Constants.EdgeDeviceHostNameKey);
+    //
+    //             if (!string.IsNullOrWhiteSpace(networkId))
+    //             {
+    //                 var endpointSettings = new EndpointSettings();
+    //                 if (module.Name.Equals(Core.Constants.EdgeHubModuleName, StringComparison.OrdinalIgnoreCase)
+    //                     && !string.IsNullOrWhiteSpace(edgeDeviceHostName))
+    //                 {
+    //                     endpointSettings.Aliases = new List<string>
+    //                     {
+    //                         edgeDeviceHostName
+    //                     };
+    //                 }
+    //
+    //                 IDictionary<string, EndpointSettings> endpointsConfig = new Dictionary<string, EndpointSettings>
+    //                 {
+    //                     [networkId] = endpointSettings
+    //                 };
+    //                 createOptions.NetworkingConfig = new NetworkingConfig
+    //                 {
+    //                     EndpointsConfig = endpointsConfig
+    //                 };
+    //             }
+    //         }
+    //     }
+    //
+    //     void MountSockets(IModule module, CreateContainerParameters createOptions)
```

Comment:
Is this for docker? That's why we don't need it?

Reply:
This has been removed by later changesets